packages/backend/src/api/factory.ts

```diff
@@ -47,6 +50,25 @@ export function createBackendApiClient(options: CreateBackendApiOptions) {
     emailAddresses: new EmailAddressAPI(request),
     instance: new InstanceAPI(request),
     invitations: new InvitationAPI(request),
+    // TODO: Remove this once we add a version to bapi-proxy
+    machineTokens: new MachineTokensApi(
+      buildRequest({
+        ...options,
+        apiVersion: '/',
+      }),
+    ),
+    idPOAuthAccessToken: new IdPOAuthAccessTokenApi(
+      buildRequest({
+        ...options,
+        apiVersion: '/',
+      }),
+    ),
+    apiKeys: new APIKeysAPI(
+      buildRequest({
+        ...options,
+        apiVersion: '/',
+      }),
+    ),
```

Comment:
bapi-proxy doesn't have versioning in its URLs yet. Using "/" for now to remove the default "v1" version during URL normalization. Will update once bapi-proxy adds proper versioning support.